syntax = "proto3";

package dataset;

// Response containing a batch of data
message DataBatch {
  bytes data = 1;
  repeated int32 labels = 2;
  int32 batch_index = 3;
  bool is_last_batch = 4;
}
